vm/devices/storage/disk_nvme/nvme_driver/src/tests.rs

```diff
@@ -414,3 +414,32 @@ impl<T: MmioIntercept + Send> DeviceRegisterIo for NvmeTestMapping<T> {
         self.mapping.write_u64(offset, data);
     }
 }
+
```

Comment:
This doesn't test any product code (it just uses the same format string). This doesn't add value, please delete it.